packages/oats-runtime/src/client.ts

```diff
@@ -196,7 +205,7 @@ function makeMethod(adapter: ClientAdapter, handler: server.Handler, pathParams:
       headers: safe(ctx).headers.$,
       query: safe(ctx).query.$,
       body: safe(ctx).body.$,
-      requestContext: {}
+      requestContext: { parameterised_path: handler.path }
```

Comment:
so this would be used by say axios adapter for logging? if so are we missing that bit in this pr.

```suggestion
      requestContext: { path: handler.path }
```

Reply:
>  I assume it would be something like /path/{id}

correct

Reply:
> so this would be used by say axios adapter for logging? if so are we missing that bit in this pr.

afaik we don't do any logging on metric collection in the axios/et al adapters, am I missing it?

Reply:
we are not. I was more thinking about how would one use the request ctx there. that would have made the usage a bit clearer. But if you promise that this is the best we can do then whatevs